src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Item/GetAzureRMRecoveryServicesBackupProtectableItem.cs

```diff
@@ -133,11 +149,30 @@ public override void ExecuteCmdlet()
 
                 if (ParameterSetName == FilterParamSet)
                 {
-                    string protectableItemType = ItemType.ToString();
-                    itemModels = itemModels.Where(itemModel =>
+                    if (ItemType != 0)
```

Comment:
@siddharth7 any reason you're not comparing by enum value, but rather integer value?

Reply:
Nothing specific here, followed the way it was being done in the same file